Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -42,9 +42,18 @@ extension BrowserViewController {
   }
   
   fileprivate func handleExternalURL(_ url: URL,
-                                     isMainFrame: Bool,
+                                     tab: Tab?,
```

Comment:
Do we only call `handleExternalURL` for specific schemes? I tried some iOS built-in schemes, such as `shortcuts://` and `shareddocuments://`, those work in Safari but not Brave. Are we intentionally don't support it?

Reply:
I could fix this but I'm not sure if I should.
`handleExternalURL` is only called when a `page` tries to open an external URL or the user presses on a link and it attempts to open an external URL.

However, when you `type` into the URL-Bar/OmniBox, only `submitSearchText` is called which processes the query and determines if to send it to the default search engine, OR if to attempt to resolve it directly. Atm, this does NOT handle external URLs, and I'm not sure if it should. 

The reason I'm not sure is because Chrome and Firefox also do not handle this case. It only works in Safari.